lib/simple_abs.rb

```diff
@@ -47,7 +47,7 @@ def ab_test_converted!(name, conversion: 1, finished: true)
       ab_test_name = ('ab_test_' + name).to_sym
       test_json = ab_test_read_cookie(ab_test_name)
       if test_json && !test_json[:finished]
-        test_record = test_json[:id] && AbTest.find_by_id(test_json[:id])
+        test_record = test_json[:id] && AbTest.find_by(id: test_json[:id], experiment: name)
```

Comment:
Instead of trusting db, we're trust cookie here. However, cookie can also be wrong. Perhaps we should have a final-level sanity check to see if tests.include?(test_value)?